Mocky/Templates/Mock.swifttemplate

```diff
@@ -315,7 +315,7 @@ func stubProperty(_ variable: SourceryRuntime.Variable, _ scope: String) -> Stri
     let aProtocol = type
      -%>
 <%_ %>// MARK: - <%= type.name %>
-class <%= type.name %>Mock: <%= type.name %>, Mock {
+class <%= type.name %>Mock:<%= type.annotations["ObjcProtocol"] != nil ? " NSObject," : "" %> <%= type.name %>, Mock {
```

Comment:
it probably would be better to have local variable like: isObjectiveCProtocol, first checking Type for objc, then annotation.